shooter/shooterapi/Shooter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace shooter.shooterapi 
+{
+    public class Shooter 
+    {
+        private readonly string name;
+        
+        private readonly int id;
+
+        public Shooter(int id, string name)
+        {
+            this.id = id;
+            this.name = name;
+        }
+        
+        public string Name
```

Comment:
you could write this as `public string name { get; };`

Reply:
According to the compiler you can't do that - 
`Shooter.cs(15,32): error CS0106: The modifier 'readonly' is not valid for this item [/home/sarge/Development/Fun_Things/shooter/shooterapi/shooterapi.csproj]`